internal/host/validator.go

```diff
@@ -1133,11 +1149,10 @@ func (v *validator) hasDefaultRoute(c *validationContext) ValidationStatus {
 		return ValidationPending
 	}
 
-	inv, err := common.UnmarshalInventory(c.host.Inventory)
-	if err != nil || len(inv.Routes) == 0 {
+	if len(c.inventory.Routes) == 0 {
```

Comment:
When does `c.inventory` get set? Are we sure it's set before this function is called?
Should this be using `c.getInventory(c.host)`?

It's a bit confusing to have an inventory cache and also a separate cache for the specific host's inventory. Maybe just remove the `c.inventory` field everywhere in favor of a helper that does `c.getInventory(c.host)`?

Reply:
It is loaded Here 
https://github.com/openshift/assisted-service/blob/dbe941db78105d7cbdf827a6b1400686f4b0fe9a/internal/host/validator.go#L192
or here
https://github.com/openshift/assisted-service/blob/dbe941db78105d7cbdf827a6b1400686f4b0fe9a/internal/host/validator.go#L217

Fixed to the format that exists in other places in the code like:
https://github.com/openshift/assisted-service/blob/dbe941db78105d7cbdf827a6b1400686f4b0fe9a/internal/host/validator.go#L566